ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockTransactionSelector.java

```diff
@@ -472,10 +465,6 @@ private boolean isTransientValidationError(final TransactionInvalidReason invali
         || invalidReason.equals(TransactionInvalidReason.NONCE_TOO_HIGH);
   }
 
-  private boolean isIncorrectNonce(final ValidationResult<TransactionInvalidReason> result) {
```

Comment:
Why is this being removed? Is the functionality happening elsewhere?

Reply:
yes it has been generalized, now every invalid reason is recorded in the `TransactionSelectionResults.invalidTransactions` map, not only for incorrect nonce, and the caller have full access to all invalid transactions